### ISSUE TITLE
Endportal failes to teleport you to the END

### ISSUE THREAD

**Issue by mormoon** [Zrips/CMI#161]
For the past few hours I've been troubleshooting why end-portals do not work anymore. After some extensive troubleshooting I have found that build of paper-spigot[1360](https://ci.destroystokyo.com/job/Paper/1360/) and onward prevent traveling to the end dimension when you have set PortalCreation: **true** in your config.yml. When it is set to **false**, you can travel to the end again. When you try an earlier build (1359) for example, the issue is resolved.  

The commit;
https://github.com/PaperMC/Paper/commit/9aa5d9ec9bd0aad081bca420cce5d6098838bd77

**Comment by LogGits** [comment]
I am not able to reproduce this. Are you using any type of multi-world manager? Is the account testing this opped? It could help if you could send in a video of the issue so I can accurately reproduce. Another big factor is that `PreventCreation` is only for Nether portals and essentially prevents players from creating nether portals by cancelling the event. Hence, I am not too sure as to why that would be the case.

**Comment by mormoon** [comment]
I can reproduce this with 3 plugins installed (PermissionsEX, CMI and ProtocolLib) The account I tested with is not opped. 

Here is a video of what's going on.
https://gfycat.com/JampackedLegalDromedary

**Comment by Zrips** [comment]
@mormoon 
You originally said "PortalCreation" but i think we are talking about "PreventCreation"?
Next, tested with same paperspigot, everything works as it should.
Next, that "PreventCreation" only effects nether portal creation and not teleportation it self to nether, no clue how it could even be connected to end portal.
Next, would be nice to have config.yml to test out if some configuration is messed up.
Next, test out with user who doesn't have any permission node. Like none at all.
Next, why you still have protocollib? CMI or PEX is not dependent from it, so you can test it without PL.

**Comment by LogGits** [comment]
Can confirm I am also having issues with cmi and world management. Regardless of the state of prevent creation.

Steps to reproduce. I used `/mvc world2 normal` (multiverse command)
then I stopped the server, then i went into server.properties and changed level-name to world2. 
After this I restarted again and the it generated `world2_nether` and `world2_the_end`. When I create a nether portal then set `preventcreation` is set to true i cant go through the portal (as expected). However, I then noticed that i couldn't go through my end portal. I tried recreating the end portal etc, nothing worked. I was unsure if what plugin it was, though I have my suspicions that it was Multiverse. I then typed `/plugman unload cmi` and i could go through the end portal. Ever since that unloading of the plugin I can go through the end portal. This is an odd issue, I think its a rare issue, nonetheless, still a bug.

**Comment by mormoon** [comment]
Thank you for confirming the issue. After your first message, I tried reproducing it on a "clean" server, but I could not reproduce it. Then I installed multiverse again, wich at first I didn't think that could be the issue, because I stopped the server completely, removed the plugin and started the server back up, and the problem was still there. However.... to reproduce this issue, you'll need to create a new world. 

So to summarize, you need paper-spigot 1360+ and multiverse to introduce the bug. 

If it helps, I can put my "test-server" on wetransfer, pm me pls.

**Comment by LogGits** [comment]
Upload a zip 👍

Another thing @Zrips, for a non opped player, when /cmi reload doesnt update PreventCreation correctly. If you start the server with prevent creation set to true and change it to false then `/cmi reload` the game wont let players enter premade portals unless its restarted.

**Comment by mormoon** [comment]
The upload is limited to 10Mb. And besides, uploading non opensource plugins to the public domain would not be a good idea in general ;) Please reach out to me in a PM, if you still want a copy op the test server.

**Comment by mormoon** [comment]
I tested the new version, but I can still reproduce this issue.  I did a drop in replacement and booted the testworld I was testing with earlier and I could not travel to the end dimension.
